google-cloud-spanner/clirr-ignored-differences.xml

```diff
@@ -631,4 +631,11 @@
     <className>com/google/cloud/spanner/connection/Connection</className>
     <method>void setDirectedRead(com.google.spanner.v1.DirectedReadOptions)</method>
   </difference>
+
+  <difference>
```

Comment:
We cannot make this change, as it breaks binary compatibility of the client library.

Reply:
Thanks for pointing this out! Reverted this change.